src/Workspaces/Core/Portable/Editing/SyntaxGenerator.cs

```diff
@@ -1868,6 +1868,10 @@ public SyntaxNode TupleElementExpression(ITypeSymbol type, string name = null)
         /// </summary>
         public abstract SyntaxNode BitwiseOrExpression(SyntaxNode left, SyntaxNode right);
 
+        internal abstract SyntaxNode ExclusiveOrExpression(SyntaxNode left, SyntaxNode right);
+        internal abstract SyntaxNode LeftShiftExpression(SyntaxNode left, SyntaxNode right);
+        internal abstract SyntaxNode RightShiftExpression(SyntaxNode left, SyntaxNode right);
```

Comment:
note: i cannot think of any reason why these should not be exposed publicly.  They make sense for both C# and VB.  @mattwar Were these intentionally not included?  If so, why?    if not, then i think we should just make public.